### DISCUSSION TITLE
Why not the official docker image?

### DISCUSSION THREAD

**Discussion by benjick** [sinamics/ztnet#156] [Q&A] [answered]
Hello!

This looks like a great project, thanks for creating it. I just wanted to ask, why not use the official zerotier docker image?

Cheers

**Comment by sinamics** [comment, marked as answer]
Hello,
Initially, I chose [zyclonite](https://github.com/zyclonite/zerotier-docker) for this project as it best met my application's needs. However, I am open to transitioning to the official version if it offers the same functionalities.

**Reply by benjick** [reply]
Makes sense, just wanted to ask 👍 thanks again!
